test/index.test.js

```diff
@@ -105,7 +105,7 @@ describe('index', () => {
             prefix: 'beta_',
             launchVersion: 'v1.2.3'
         });
-        assert.equal(executor.buildTimeout, 3600);
+        assert.equal(executor.buildTimeout, 30);
```

Comment:
Same as above ^

Reply:
Arbitrary value. It's to test that the object is constructed with whatever value was passed in for `buildTimeout`

Reply:
Cool, approved!